### ISSUE TITLE
progress bar error

### ISSUE THREAD

**Issue by mcfanda** [r-lib/gh#114]
Hello
I'm using gh v 1.1.0. In the gh() function, the option .progress=FALSE gives always an error

`Error in update_progress_bar(prbr, res) : object 'prbr' not found`

**Comment by hadley** [comment]
``` r
x <- gh::gh(
  "GET /repos/:owner/:repo/releases",
  owner = "gohugoio",
  repo = "hugo",
  .limit = Inf,
  .progress = FALSE
)
#> Error in update_progress_bar(prbr, res): object 'prbr' not found
```

<sup>Created on 2020-06-04 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by gaborcsardi** [comment]
This was fixed in #115
